node/runtime/src/impls.rs

```diff
@@ -49,10 +49,17 @@ impl Convert<u128, Balance> for CurrencyToVoteHandler {
 	fn convert(x: u128) -> Balance { x * Self::factor() }
 }
 
-/// Simply multiply by a coefficient denoted by the `Get` implementation.
-impl Convert<Weight, Balance> for WeightToFee {
-	fn convert(x: Weight) -> Balance {
-		Balance::from(x).saturating_mul(Self::get())
+// M is the coefficient in units of currency / weight
```

Comment:
```suggestion
/// M is the coefficient in units of currency / weight
```

and it should be a proper doc of what it is